sonic-stormond/scripts/stormond

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python3
+
+"""
+    stormond
+    Storage device Monitoring daemon for SONiC
+"""
+
+import os
+import signal
+import sys
+import threading
+import subprocess
+import shutil
+import json
+import time
+
+from sonic_py_common import daemon_base, device_info, logger
+from swsscommon import swsscommon
+from sonic_platform_base.sonic_storage.storage_devices import StorageDevices, BLKDEV_BASE_PATH
+
+#
+# Constants ====================================================================
+#
+
+# TODO: Once we no longer support Python 2, we can eliminate this and get the
+# name using the 'name' field (e.g., `signal.SIGINT.name`) starting with Python 3.5
+SIGNALS_TO_NAMES_DICT = dict((getattr(signal, n), n)
+                             for n in dir(signal) if n.startswith('SIG') and '_' not in n)
+
+SYSLOG_IDENTIFIER = "stormond"
+
+STORAGE_DEVICE_TABLE = "STORAGE_INFO"
+FSSTATS_SYNC_TIME_KEY = "FSSTATS_SYNC"
+
+FSIO_RW_JSON_FILE = "/usr/share/stormond/fsio-rw-stats.json"
+
+STORMOND_MAIN_THREAD_SLEEP_SECS = 3600 #one hour
+STORMOND_SYNC_TO_DISK_SECS = 86400 #one day
+
+STORAGEUTIL_LOAD_ERROR = 127
+
+log = logger.Logger(SYSLOG_IDENTIFIER)
```

Comment:
@assrinivasan please use SysLogger as direct use of logger pollutes the identifier with other daemon's identifier

Reply:
will do!

Reply:
Added in latest